exercise-src/react-native/05-intro-to-testing/01-problem/App.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from "@testing-library/react-native"
+import App from "./App"
+
+describe("App Component", () => {
+  it("renders correctly with given props", () => {
+    throw new Error("Test needs to be implemented")
+  })
+})
```

Comment:
Did you talk to @christopherjbaker about these exercises? I want to make sure you did because they’re brand new.

Reply:
Turns out there will be no exercises for this section.